server/src/push/emitters.js

```diff
@@ -58,6 +59,31 @@ const getPushEmitters = ({ models }) => {
         });
       });
     },
+    pushMessage({ message, eventId }) {
+      if (eventId) {
+        // // TODO: Single joined query
+        Event.findById(eventId).then((event) => {
```

Comment:
need to return the promise or use async/await

Reply:
how come? I dont care about the result I just fire it off and assume it finishes OK

Reply:
i believe it's important to present it to the caller. if the caller chooses not to block waiting for it that is fine.

Reply:
none of the other emitters return. should i fix them all in one hit in a seperate PR?

Reply:
ideally, yes. fine to leave this as is for now